src/training_hub/algorithms/its_rollout.py

```
@@ -0,0 +1,213 @@
+
+        except (ConnectionError, TimeoutError) as e:
+            logger.warning("ITS Hub rollout failed (retriable): %s", e)
+            fallback_choice = Choice(
+                index=0,
+                finish_reason="stop",
+                message=ChatCompletionMessage(
+                    role="assistant", content="[rollout failed]"
+                ),
+            )
+            trajectory = art.Trajectory(
+                messages_and_choices=list(messages) + [fallback_choice],
```

Ruff (0.15.15)

[warning] Consider `[*list(messages), fallback_choice]` instead of concatenation

Replace with `[*list(messages), fallback_choice]`

(RUF005)